dbms/src/Flash/Coprocessor/JoinInterpreterHelper.cpp

```diff
@@ -190,40 +190,38 @@ TiDB::TiDBCollators getJoinKeyCollators(const tipb::Join & join, const JoinKeyTy
     return collators;
 }
 
-JoinOtherConditions doGenJoinOtherConditionsAction(
+void doFillJoinOtherConditionsAction(
```

Comment:
how about moving to `JoinNonEqualConditions::fillJoinOtherConditionsAction`?

Reply:
Done